db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -243,16 +243,19 @@ Status DBImpl::FlushMemTableToOutputFile(
   // If the log sync failed, we do not need to pick memtable. Otherwise,
   // num_flush_not_started_ needs to be rollback.
   TEST_SYNC_POINT("DBImpl::FlushMemTableToOutputFile:BeforePickMemtables");
+  SequenceNumber earliest_seqno = kMaxSequenceNumber;
+  SequenceNumber largest_seqno = 0;
```

Comment:
How about both use 0 for uninitialized? It would be unnatural to have `earliest_seqno > largest_seqno`.